stdlib/public/TensorFlow/DataTypes.swift

```diff
@@ -34,6 +34,40 @@ public struct TensorDataType {
   internal init(_ cDataType: TF_DataType) {
     self._cDataType = cDataType
   }
+
+  @usableFromInline
+  internal func makeTensor(owning pointer: CTensorHandle) -> AnyTensor {
```

Comment:
This also feels a little awkward. Calling a method on a *dtype* that takes a pointer feels a little backwards. Why not just define a top level function ‘makeTensor(owningHandle:dataType:)’?

It should also be possible if the data type is TF_STRING or TF_RESOURCE. Why make them fatal error?

Reply:
No conformance to AnyTensor.